tests/testthat/test-xml_find.R

```diff
@@ -85,6 +85,26 @@ test_that("xml_find_num returns a numeric result", {
   expect_equal(xml_find_num(x, "1 div floor(-0)"), -Inf)
 })
 
+# Find int ---------------------------------------------------------------------
+test_that("xml_find_int errors with non integer results", {
+  x <- read_xml("<x><y/><y/></x>")
+  expect_snapshot_error({
```

Comment:
We now prefer `expect_snapshot()` + `error = TRUE` so then you see the code and the error in the snapshot.

Reply:
#414. Would you prefer fixing that first, or as a follow-up?

Reply:
Fix it here, and then we'll update the other tests in a later PR.